src/lib/alarms/trade_offer.ts

```diff
@@ -222,6 +248,30 @@ function offerStateMapper(e: TradeOffersAPIOffer): OfferStatus {
     } as OfferStatus;
 }
 
+function extendedOfferStateMapper(e: TradeOffersAPIOffer) {
```

Comment:
Is there actually a need for this transform? It doesn't seem to provide value beyond processing steamid32 to steamid64 (which we can get the caller do to if they need it)